tests/acceptance/features/webUITags/createTags.feature

```diff
@@ -67,6 +77,8 @@ Feature: Creation of tags for the files and folders
 
   @skip @yetToImplement
   Scenario: Delete a tag in a shared file
+    Given user "Alice" has created a new share with following settings
```

Comment:
`following settings` are missing
Even though the tests are skipped you  can add the data table